runtime-sdk/src/types/transaction.rs

```diff
@@ -7,7 +7,9 @@ use crate::{
         multisig,
         signature::{self, PublicKey, Signature},
     },
+    modules::core::types::CallerAddress,
```

Comment:
Should this be moved to the general types (e.g. this module) instead since it is used here?